.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+updates:
+# Enable version updates for Maven
+- package-ecosystem: "maven"
+  # Look for a `pom.xml` in the `root` directory
+  directory: "/"
+  # Check for updates every day (weekdays)
+  schedule:
+    interval: "daily"
+    # Check for Maven updates at 4am UTC
+    time: "04:00"
+    timezone: "Europe/Lisbon"
+  open-pull-requests-limit: 1
```

Comment:
Do we really want to limit this?

Reply:
We have to put a limit, otherwise it will probably create many pull requests at once.
Maybe just 1 is too low, and we can raise it to like 3 or 4 at a time.

Reply:
Yes, I agree with having a limit. Any small number (e.g. between 1 and 5) should be ok for now. And we can tweak it later.

Reply:
Let's use 3 as a rule of thumb and manage accordingly later. WDYT?

Reply:
Agreed